public/locales/pt/affix.json

```diff
@@ -1,6 +1,6 @@
 {
-    "affixButtonTitle": "Clique aqui para adicionar um discurso ou personalidade",
-    "affixButtonCreateClaim": "Clique aqui para adicionar um discurso",
+    "affixButtonTitle": "Clique aqui para adicionar uma afirmação ou personalidade",
```

Comment:
Suggestion: @thesocialdev, @Keikonichi What do y'all think of replacing it to "Clique aqui para adicionar conteudo na plataforma" ?

Reply:
The word 'content' seems too broad; a more specific and formal alternative could be replacing it with 'information' or 'data.'

This way, it would be: 'Clique aqui para adicionar informações à plataforma' or 'Clique aqui para inserir dados na plataforma.'

These terms align better with the text presented in the modal the first time a user logs in.

Reply:
"Afirmação"/Claim are better representing our intentions Today.